_posts/2020-03-24-dask-developer-workshop.markdown

```diff
@@ -0,0 +1,90 @@
+---
+layout: single
+title:  "Karlsruhe to D.C. ― a Dask story"
+date:   2020-03-24 20:00:00 +0100
+tags: technology python data-engineering Dask distributed
+header:
+  overlay_image: assets/images/dask_icon.svg
+  overlay_filter: 0.2
+  show_overlay_excerpt: false
+author: Lucas Rademaker & Nefta Kanilmaz
+author_profile: true
+---
+# Karlsruhe to D.C. ― a Dask story
+
+Back in February 2020, we (Florian Jetter, Nefta Kanilmaz and Lucas Rademaker) travelled to the Washington D.C. metropolitan area to attend the first Dask developer conference. Our primary goal was to discuss Blue Yonder’s issues related to Dask with the attending developers and users. When we returned back to Germany, we had not only connected with many of the core developers in the Dask community, but also had an (almost finished) implementation of a distributed semaphore in our bags. 
+
+There is [a blog post](https://blog.dask.org/2020/04/28/dask-summit) giving an overview on and summary of the workshop talks. We want to share our experience around this workshop. 
+
+## Setting
+
+The three-day long workshop included sessions with short talks, as well as time slots for working on Dask-related issues and discussions.
+Talks covered a broad spectrum of topics including Dask-ML and usage of Dask in general data analysis, Dask deployment and infrastructure, and many more. All talks focused on answering these key questions: 
```

Comment:
"The talks covered... and the usage of Dask... Dask deployment ~~and~~ & infrastructure ~~,~~ [no comma before and] and many more. All the talk seemed to focus on answering ~~these~~ three key questions:"

```suggestion
The talks covered a broad spectrum of topics including Dask-ML and the usage of Dask in general data analysis, Dask deployment & infrastructure and many more. All talks seemed to focus on answering three key questions: 
```

Reply:
why `&` over `and`?